### PR TITLE
Explain how to handle terminated services and how to rename services

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:516]:
```suggestion
### Handling terminated services
```

or 

```suggestion
### Handling a terminated service
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:518]:
```suggestion
If the service provider stops offering a service, the associated terms will become unavailable. To mark that service termination in Open Terms Archive, move all the terms of the service to its history:
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:520]:
```suggestion
1. Update the declaration to stop tracking all terms, by removing all the `<terms type>` entries from the  `documents` key in the declaration:
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:528]:
```suggestion
2. Move the documents declaration to the service [history file](#terms-declaration-history).
```

Comment (MattiSG) [inline content/contributing-terms.en.md:528]: What should the `validUntil` date be? 🙂

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: content/contributing-terms.en.md
Comment (clementbiron) [reply content/contributing-terms.en.md]: I guess what's already written is enough.

> Both for terms and filters history, the expiration date is declared in a property `validUntil`. It should be the authored date and time of the last snapshot commit for which the declaration is still valid.

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:521]: > I believe there needs to be no empty line for Markdown to render this as a list 🤔 
```suggestion
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:522]:
```suggestion
2. Update the declaration to stop tracking all terms, by removing every `<terms type>` entries from the  `documents` key in the declaration:
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:518]:
```suggestion
If the service provider stops offering a service, the associated terms will become unavailable. To mark that service termination in Open Terms Archive and ensure tracking tentatives are stopped, while maintaining the possibility to explore the history:
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:520]:
```suggestion
1. Move the existing documents declaration to the service [history file](#terms-declaration-history).
```

File: content/contributing-terms.en.md
Comment (MattiSG) [inline content/contributing-terms.en.md:532]: Avoid usage of unclear “we”.
```suggestion
The consensus is to consider that a service provider renaming a service (for example, `Twitter` to `X`) is akin to terminating the previous service and opening a new one. Therefore, to apply a service renaming, open a pull request that both [terminates the previous service](#handling-terminated-service) and adds a new [service declaration](#declaring-a-new-service) with the new service name. You can reuse the `documents` part of the original declaration, but should double-check that the selectors and URLs still match, as a service rename is most often accompanied by a new page layout, a new domain name, and sometimes entirely new terms.
```

**Review by MattiSG — APPROVED** [review]
Great!

**Comment by MattiSG** [comment]
Please notify https://github.com/OpenTermsArchive/contrib-declarations/pull/906 once merged!

**Review by pg-adrian — COMMENTED** [review]
Hi all, thank you for including me in this! I have a question regarding considering "renaming a service as akin to opening a new one". 

In the case of Twitter/X we see that it really mostly is renaming, so the Twitter policies are mostly continued and just the "Twitter" in the policies is replaced with "X". So by terminating the existing service and opening a new one for a renaming, are we not blurring this continuity for users of the data?

**Comment by MattiSG** [comment]
Through discussion, we reached a conclusion that it was more relevant to leave it to the analysts / consumers to decide how to join data. It is rather trivial, at analysis stage, to fuse `Twitter` and `X` from the dataset: just merge the folders.
However, if we use only one name, we make it hard to discover the data: will users expect `X` to contain historical 2011 data of `Twitter`? If we keep the original name, will it still make sense in 5 years when nobody remembers anymore that `Make` used to be called `Integromat`?

Now, there might be a case for adding metadata on renames. At this stage, we recommend doing in at commit level, by having a single commit with message “Rename Twitter to X” that contains both the termination of Twitter and the tracking of X.

This example is especially interesting since we observe that the transition of terms is very gradual: while one can pinpoint the official renaming date announcement, or the incorporated name change, it is impossible to pinpoint that transition in the terms as the renaming is applied apparently opportunistically, with terms still being rewritten in the last weeks, several months after the official name change.

The policy we described here is the current best practice, that does not mean it is final 🙂 We would happily consider improvements or complete changes if there are clear use cases and applicable proposals!

**Pull request merged**
